custom/icds/location_reassignment/models.py

```diff
@@ -69,6 +74,20 @@ def valid(self):
                 self.errors.append("%s operation: location %s with site code %s is already deprecated." % (
                     self.type, new_location.name, new_location.site_code))
                 valid = False
+        valid = valid and self._validate_location_count(self.expected_old_locations,
+                                                        len(self.old_locations), "old")
```

Comment:
Might as well make "old" and "new" constants, too.

Reply:
do you have suggestions for better names, i have used
source destination
old new..
but not happy with either.

Reply:
I think old and new are all right. The only other thing I can think of is before/after, but I don't think that's obviously better.

Reply:
that probably isnt right considering the variable names have the "old" and "new" as well. Dont see a benefit from making those constants. so going to leave this as-is